common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -236,10 +236,13 @@ object ContentWidths {
 
   object GalleryMedia {
     val inline = WidthsByBreakpoint(
-      mobile          = Some(445.px),
-      mobileLandscape = Some(610.px),
-      phablet =         Some(620.px),
-      tablet =          Some(700.px)) // desktop, leftCol, and wide are also 700px
+      mobile          = Some(480.px),
+      mobileLandscape = Some(660.px),
+      phablet =         Some(700.px),
+      tablet =          Some(700.px), // TODO: Change to 480 when new galleries is merged
+      desktop =         Some(720.px),
+      leftCol =         Some(880.px),
+      wide =            Some(1010.px))
```

Comment:
This is displayed at 1040px wide on widest BP at 100% forcing upsizing by the browser which, in turn, reduces quality and introduces interpolation artifacts. Other sizes are also not exactly as they appear on page. BTW, I wonder how this works for portrait images that are now shown fluidly at the viewport height...